docs/getting_started.md

```diff
@@ -1,154 +1,129 @@
+vkObject object = wrapper.object;
 ```
 
-To query the available layers and extensions, get a `SystemInfo` struct from `SystemInfo::get_system_info()`. It contains a `is_layer_available()` and `is_extension_available()` function to check for a layer or extensions before enabling it. It also has booleans for if the validation layers are present and if the VK_EXT_debug_utils extension is available.
+# Instance Creation
+
+Taking a look at a simple example
 
 ```cpp
-auto system_info_ret = vkb::SystemInfo::get_system_info();
-if (!system_info_ret) {
-    std::cerr << system_info_ret.error().message().c_str() << "\n";
+vkb::InstanceBuilder instance_builder;
+auto instance_ret = instance_builder
+                            .set_app_name("Awesome Vulkan Application")
+                            .set_engine_name("Excellent Game Engine")
+                            .require_api_version(1,0,0)
+                            .build(); // build is always called last
+
+// simple error checking and helpful error messages
+if (!instance_ret) {
+    std::cerr << "Failed to create Vulkan instance. Error: " << instance_builder_return.error().message() << "\n";
     return -1;
 }
-auto system_info = system_info_ret.value();
-if (system_info.is_layer_available("VK_LAYER_LUNARG_api_dump")) {
-    instance_builder.enable_layer("VK_LAYER_LUNARG_api_dump");
-}
-if (system_info.validation_layers_available){
-    instance_builder.enable_validation_layers();
-}
-```
 
-The `vkb::Instance` struct is meant to hold all the necessary instance level data to enable proper Physical Device selection. It also is meant for easy destructuring into custom classes if so desired.
-```cpp
-struct CustomVulkanWrapper {
-    VkInstance instance;
-    //...
-};
-CustomVulkanWrapper custom_vk_class;
-custom_vk_class.instance = vkb_instance.instance;
+// Get handle and use however you want!
+VkInstance instance = instance_ret.value();
 ```
 
-When the application is finished with the vulkan, call `vkb::destroy_instance()` to dispose of the instance and associated data.
-```cpp
-// cleanup
-vkb::destroy_instance(vkb_instance);
-```
-### Instance Creation Summary
+## Versions
+
+There are many "versions" in the world of Vulkan and `InstanceBuilder` lets you set them all!
+
 ```cpp
 vkb::InstanceBuilder instance_builder;
-auto instance_builder_return = instance_builder
-        // Instance creation configuration
-        .request_validation_layers()
-        .use_default_debug_messenger()
-        .build ();
-if (!instance_builder_return) {
-    // Handle error
-}
-vkb::Instance vkb_instance = instance_builder_return.value ();
 
-// at program end
-vkb::destroy_instance(vkb_instance);
+// Sets VkApplicationInfo::applicationVersion
+instance_builder.set_app_version(3, 0, 0);
+// Sets VkApplicationInfo::engineVersion
+instance_builder.set_engine_version(5, 0, 2);
+
+// The Instance Version (aka the version of your loader)
+// TODO - which?
```

Comment:
@charles-lunarg

Reply:
Dug through the source code and came up with an answer:

* `require_api_version` says that "1.X must be supported, and set the apiVersion to 1.X"
* `set_minimum_instance_version` says that "1.X must be supported, but don't change the apiVersion.

If you are looking at that and going "wait what, why?". You are thinking what I'm thinking.

I can't see a particular reason to want a high API version unless you specify that you are going to use it. Otherwise, just use the lower version man.

I think I'll make a PR to deprecate it.

Reply:
(aka, if this is good otherwise, just get this in)

Reply:
Yeah, I'll merge then fixup.